### ISSUE TITLE
using another database server

### ISSUE THREAD

**Issue by ccfiel** [skool51/shellyjs#5]
I have notice its using sqlite for database. Can I change to other more scalable  database server?

**Comment by sdale511** [comment]
Yes it supports Redis now and has a plugin structure for the DB.

sqlite is just for quick and dirty dev.

http://www.shellyjs.com/learn.html#useRedis

-- 
Scott Dale

On January 9, 2015 at 4:18:18 PM, Chris Ian Fiel (notifications@github.com) wrote:

I have notice its using sqlite for database. Can I change to other more scalable database server?
